package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "eth-packages",
+  "license": "MIT",
+  "scripts": {
+    "format:check": "prettier --check .",
+    "lint": "eslint --cache .",
+    "test": "npx lerna run test --concurrency 1 --stream",
+    "test:changed": "npm run test -- --since master"
+  },
+  "dependencies": {
+    "lerna": "^7.0.2"
+  },
+  "devDependencies": {
+    "@types/chai-as-promised": "7.1.8",
+    "@types/mocha": "10.0.6",
+    "@types/sinon": "17.0.3",
+    "@typescript-eslint/eslint-plugin": "7.8.0",
+    "@typescript-eslint/parser": "7.8.0",
+    "chai": "4.4.1",
+    "chai-as-promised": "7.1.1",
+    "eslint": "^8.43.0",
+    "eslint-config-bloq": "^4.0.2",
+    "husky": "^4.3.8",
+    "lint-staged": "^13.2.2",
+    "mocha": "10.4.0",
+    "prettier": "^3.2.5",
+    "sinon": "17.0.1"
+  },
+  "workspaces": [
+    "packages/*"
+  ],
+  "eslintConfig": {
+    "extends": [
+      "bloq",
+      "plugin:@typescript-eslint/eslint-recommended",
+      "plugin:@typescript-eslint/recommended",
```

Comment:
Should we add these to `eslint-config-bloq`'s default ruleset?

Reply:
we could (outside of this scope).
We would need something like `bloq/typescript` I believe, because we should use it for TS files only

Reply:
Created https://github.com/bloq/eslint-config-bloq/issues/44